### PR TITLE
Dataset view

### PR DESCRIPTION
**Pull request by xavijam**
First step with dataset view.

- Render table name + privacy.
- Readonly for sync tables. 
- Render table view.
- Reviewed in several browsers.

screen shot 2016-06-29 at 12 08 57

Part of: https://github.com/CartoDB/cartodb/issues/7950

CR: @nobuti

### CODE REVIEW

File: lib/assets/javascripts/cartodb3/dataset/header-view.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+var CoreView = require('backbone/core-view');
+var template = require('./header.tpl');
+var moment = require('moment');
+var title = 'CartoDB';
```
Comment (nobuti) [inline lib/assets/javascripts/cartodb3/dataset/header-view.js]: DB?

File: lib/assets/javascripts/cartodb3/dataset/header-view.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        ago: moment(this._tableModel.get('updated_at')).fromNow()
+      })
+    );
+    return this;
+  },
+
+  _initBinds: function () {
+    this._tableModel.bind('change', this.render, this);
+    this._tableModel.bind('change:name', this._onChangeName, this);
+    this._visModel.bind('change', this.render, this);
```
Comment (nobuti) [inline lib/assets/javascripts/cartodb3/dataset/header-view.js]: add_related_models

File: app/assets/stylesheets/editor-3/dataset.css.scss [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+.Dataset {
+  @include display-flex();
+  @include flex-direction(column);
+  @include flex(1);
+}
+.Dataset-view {
+  @include display-flex();
+  @include flex-direction(column);
+  position: relative;
+  width: 100%;
```
Comment (nobuti) [inline app/assets/stylesheets/editor-3/dataset.css.scss]: do we need this?

**Comment by nobuti** [comment]
👍

File: app/assets/stylesheets/editor-3/dataset.css.scss (hunk 3 above)
Comment (xavijam) [inline app/assets/stylesheets/editor-3/dataset.css.scss]: True, thanks! :)

**Pull request merged**